pkg/domain/infra/abi/containers.go

```diff
@@ -1361,7 +1362,11 @@ func (ic *ContainerEngine) ContainerInit(ctx context.Context, namesOrIds []strin
 }
 
 func (ic *ContainerEngine) ContainerMount(ctx context.Context, nameOrIDs []string, options entities.ContainerMountOptions) ([]*entities.ContainerMountReport, error) {
-	if os.Geteuid() != 0 {
+	hasCapSysAdmin, err := unshare.HasCapSysAdmin()
```

Comment:
Can this error happen without something critical going wrong?

Reply:
we have this function called in different places, and the result is cached.  So if it fails, even if we ignore it here, it will be caught somewhere else